shared/constants/chat2/convostate.tsx

```diff
@@ -2581,9 +2581,19 @@ const createSlice: Z.ImmerStateCreator<ConvoState> = (set, get) => {
     setMeta: _m => {
       // see updatemeta
       const m = _m ?? Meta.makeConversationMeta()
+      const wasGood = get().isMetaGood()
       set(s => {
         C.updateImmer(s.meta, m)
       })
+      const isGood = get().isMetaGood()
```

Comment:
when we get a real meta, adopt the draft as the unsentText once ever if there's nothing to overwrite